src/Altinn.Apps/AppFrontend/react/altinn-app-frontend/src/types/index.ts

```diff
@@ -175,7 +175,7 @@ export interface INavigationConfig {
 
 export interface IOption {
   label: string;
-  value: any;
+  value: string;
```

Comment:
@lorang92 
Do we need to support numbers?

Reply:
It's a string on the server [/src/Altinn.Apps/AppTemplates/AspNet/Altinn.App.Common/Models/AppOption.cs#L11](https://github.com/altinn/altinn-studio/blob/75635e679770d29167f9d26fce37cd73eab54713/src/Altinn.Apps/AppTemplates/AspNet/Altinn.App.Common/Models/AppOption.cs#L11)